### DISCUSSION TITLE
SNS Option in Elastalert of Security Onion

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#8231] [Unsupported Versions]
Hi, 
Are there any plans to have SNS in elastalert of Security Onion? Elastalert 2 already has SNS feature and I was just wondering if it would be brought in Security Onion.

**Comment by weslambert** [comment]
The SNS alerter for Elastalert should already be available in the latest version of Security Onion.

For more information on the alerter, see:
https://elastalert.readthedocs.io/en/latest/ruletypes.html#sns
